flake.nix

```diff
@@ -0,0 +1,41 @@
+{
+  description = "Cubical Agda";
+
+  inputs.flake-utils.url = github:numtide/flake-utils;
+
+  outputs = { self, flake-utils, nixpkgs }:
+    flake-utils.lib.eachDefaultSystem (system:
+    let pkgs = import nixpkgs { inherit system; };
+        cubical = with pkgs;
+          agdaPackages.mkDerivation rec {
+            pname = "cubical";
+            version = "0.4";
+
+            src = pkgs.lib.cleanSourceWith {
+              filter = name: type:
+                !(pkgs.lib.hasSuffix ".nix" name)
+              ;
+              src = ./.;
+            };
+
+
+            LC_ALL = "en_US.UTF-8";
```

Comment:
`C.UTF-8` should be enough, and then we can remove `glibcLocales`.

Reply:
I put C.UTF-8

Reply:
I don't see `glibcLocales` anywhere, so I assume this can be closed.